common/src/main/java/com/alibaba/nacos/common/remote/client/RpcClientFactory.java

```diff
@@ -34,11 +34,11 @@
  * @version $Id: RpcClientFactory.java, v 0.1 2020年07月14日 3:41 PM liuzunfei Exp $
  */
 public class RpcClientFactory {
-    
+
```

Comment:
indent problem.
Please use nacos code style to reformat.

Reply:
done.